src/lib/settings/getDefaultRange.js

```diff
@@ -11,6 +18,14 @@ function calcBaseRange(s, width, height, reverse, percentRange) {
 	return reverse === true ? [max, min] : [min, max];
 }
 
+/**
+ * @param {'x'|'y'|'z'|'r'} s
+ * @param {number} width
+ * @param {number} height
+ * @param {boolean} reverse
+ * @param {[number, number]|Function|null} range
```

Comment:
Instead of function, what if we were more specific and defined the arguments. Something like this?

```js
/*
 * @typedef {function} RangeType
 * @param {{width: number, height: number}} dimensions - The dimensions object containing width and height.
 * @returns {Array<number|string>} The calculated range .
```